packages/commons/src/config/notificationConfig.ts

```diff
@@ -0,0 +1,13 @@
+import { z } from "zod";
+
+export const NotificationConfig = z
```

Comment:
this is not a shared config, may we move this to the `notifier-seeder` package?

Reply:
move to notifier-seeder package